thirdeye/thirdeye-frontend/app/pods/manage/alert/edit/controller.js

```diff
@@ -284,7 +226,7 @@ export default Controller.extend({
    */
   prepareFunctions(configGroup, newId = 0) {
     const newFunctionList = [];
-    const existingFunctionList = configGroup.emailConfig ? configGroup.emailConfig.functionIds : [];
+    const existingFunctionList = configGroup && configGroup.emailConfig ? configGroup.emailConfig.functionIds : [];
```

Comment:
fyi we can just use the _.has from lodash  https://lodash.com/docs/4.17.4#get

It's behaves similarly to `Ember.getWithDefault` but works with non Ember object

Reply:
done - thanks!